docs/examples/101/resource-with-lock-existing/main.bicep

```diff
@@ -0,0 +1,13 @@
+param accountName string
+
+resource storageAcc 'Microsoft.Storage/storageAccounts@2019-06-01' existing = {
```

Comment:
I have one question about the position of the `existing` keyword. If we are going add support to `else` for conditional resources in the future, what would it be look like if I want the `existing` keyword to target the `else` resource body?